common/src/main/java/whocraft/tardis_refined/common/block/door/InternalDoorBlock.java

```diff
@@ -96,10 +97,15 @@ public void entityInside(BlockState blockState, Level level, BlockPos blockPos,
         if (!level.isClientSide()) {
             ServerLevel serverLevel = (ServerLevel)level;
             if (serverLevel.getBlockEntity(blockPos) instanceof TardisInternalDoor door) {
-                AABB teleportAABB = this.getCollisionShape(blockState, level, blockPos, CollisionContext.of(entity)).bounds().move(blockPos);
-                if (TRTeleporter.teleportIfCollided(serverLevel, blockPos, entity, teleportAABB)){
-                    door.onAttemptEnter(blockState, serverLevel, blockPos, entity);
+
+                if (TardisLevelOperator.get(serverLevel).isPresent()) {
```

Comment:
Reason for the changes here? Nothing against them, just wondering why

Reply:
The block will teleport you to the center of the world if it's not the TARDIS dimension.